backend/docs/API_Specifications/asset-registry-1.0.0.yaml

```
@@ -0,0 +1,710 @@
+          description: Master data code identifying the type of asset
+          minLength: 2
+          maxLength: 64
+        serialNumber:
+          type: string
+          description: User entered serial number of the asset
+        modelNumber:
+          type: string
+          description: User entered model number for an asset
+        brandID:
+          type: string
+          description: Master data code of the brand from MDMS. 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)